Modules/Package.resolved

```diff
@@ -0,0 +1,14 @@
+{
+  "pins" : [
```

Comment:
The dependencies will be pinned in `Modules/Package.resovled`. 
For convenience, I could add symlinks from the root to `Modules/Package.swift`.

Reply:
I did more testing, and I think it actually works the way you'd expect. I don't know why it didn't work the first time. The dependencies from `Modules/Package.swift` are pinned in `Modules/Package.resovled` and this file does get updated even if you open an Xcode project with the app and not the package itself. The integration with Xcode is also pretty good.

It's easy to test that the pins are working:

- Open the app in Xcode
- Set Gravatar dependency to `.package(url: "https://github.com/Automattic/Gravatar-SDK-iOS", exact: "2.0.0")`
- Verify that the "Package Dependencies" now show "Gravatar 2.0.0" and that `Modules/Package.resolved` got updated with a new pin
- Change Gravatar dependency to `.package(url: "https://github.com/Automattic/Gravatar-SDK-iOS", from: "2.0.0")`
- Verify that the package doesn't get updated automatically to the latest version of Gravatar, which is 2.1.0
- Re-open Xcode and verify the previous step again
- Find Gravatar In Xcode "Project Navigator" / "Package Dependencies" and click "Update Package"
- Verify that the `Modules/Package.resovled` got updated and now points to the latest version (currently 2.1.0)

Reply:
Btw, it looks like I ended up inadvertently updating some of the dependencies, but I can revert it.